picard/ui/itemviews/__init__.py

```diff
@@ -512,13 +512,13 @@ def update(self, update_album=True, update_files=True, update_selection=True):
                 newnum = track.num_linked_files
                 if oldnum > newnum:  # remove old items
                     for i in range(oldnum - newnum):
-                        self.takeChild(newnum - 1).obj.item = None
+                        self.takeChild(newnum - 1).obj.ui_item = None
                     oldnum = newnum
                 for i in range(oldnum):  # update existing items
                     item = self.child(i)
                     file = track.files[i]
                     item.obj = file
-                    file.item = item
+                    file.ui_item = item
```

Comment:
I think we can get rid of intermediate `item` variable here:

```python
    file = track.files[i]
    file.ui_item = self.child(i)
    file.ui_item.obj = file                 
```

Reply:
I took a bit different approach in 402f8463b62a9327e03e935c28b74e39ca1c268d